panels/exo_skel.py

```diff
@@ -102,6 +104,34 @@ def execute(self, context):
                 bone_item.name = bone.name
             
         return {'FINISHED'}
+
+class UpdateBoneList(bpy.types.Operator):
+    bl_idname = 'sub.update_bone_list'
+    bl_label = 'Update Bone Pairing List'
+    bl_description = 'Updates the current pairing list where you match bones from the smash armature to the other armature'
+    bl_options = {'REGISTER', 'UNDO', 'INTERNAL'}  
+    
+    def execute(self, context):
+        context.scene.saved_bone_list.clear()
+        for bone_item in context.scene.bone_list:
+            if bone_item.bone_name_smash:
+                saved_bone_item = context.scene.saved_bone_list.add()
+                saved_bone_item.bone_name_other = bone_item.bone_name_other
+                saved_bone_item.bone_name_smash = bone_item.bone_name_smash
+
+        BuildBoneList.execute(self, context)
+
+        cur_bone_other_list = [bone_item.bone_name_other for bone_item in context.scene.bone_list]
+        
+        if not context.scene.saved_bone_list:
+            self.report({'ERROR'}, "Couldn't save current bone list because it was empty")
```

Comment:
Does this need to be an error condition? I'm only able to trigger this case when there were no bones paired prior to updating the bone list. We should be able to just skip filling in previous pairings if the list is empty.

Reply:
It doesn't really have to be an error condition. I put it there because if the user wanted to rebuild their list, and it didn't save/properly transfer over the previous pairings to the new one, they would have "some" idea of what went wrong.